test/integration/httpd_role_test.lua

```diff
@@ -3,10 +3,18 @@ local treegen = require('luatest.treegen')
 local server = require('luatest.server')
 local fun = require('fun')
 local yaml = require('yaml')
+local fio = require('fio')
+local http_client = require('http.client').new()
+
 
 local helpers = require('test.helpers')
 
-local g = t.group()
+local g = t.group(nil, {
+    { use_tls = false },
+    { use_tls = true },
+})
```

Comment:
```suggestion
local g = t.group(nil, {
    t.helpers.matrix({ use_tls = {false, true} }),
})
```

Reply:
Isn't matrix uses in case when we want to combine several params? But here we have only one.

Moreover this suggestion isn't working with only one target param: `parameter name should be string, got number`.

Reply:
> Isn't matrix uses in case when we want to combine several params? But here we have only one.

Well, it is more useful this way, but even for one parameter it seems to be a bit more readable.

> Moreover this suggestion isn't working with only one matrix param: parameter name should be string, got number.

I think you're made a mistake in array/map mixing somewhere, it works fine

```lua
tarantool> t.helpers.matrix({use_tls = {true, false}})
---
- - use_tls: true
  - use_tls: false
...

tarantool> t.helpers.matrix({use_tls = {true}})
---
- - use_tls: true
...
```

Reply:
Yes, I'm sorry, just used `matrix` already inside table which caused an error. Fixed.